src/modules/controls/components/Controls.jsx

```diff
@@ -70,11 +72,19 @@ export default function Controls({
 
 	return (
 		<KeyboardControls
+			isPlayerOpen={!!currentlyPlaying}
 			togglePlaying={togglePlaying}
 			toggleExpanded={toggleExpanded}
 			toggleSearch={toggleSearch}
 		>
-			<MediaQuery query={device.small}>{ifElse(mobile, desktop)}</MediaQuery>;
+			<VelocityTransitionGroup
```

Comment:
love how you're able to just wrap the group and then call the pre declared animations from the `animations.js` file